spec/machine/Kernel_Config_Lemmas.thy

```diff
@@ -0,0 +1,28 @@
+(*
+ * Copyright 2021, Proofcraft Pty Ltd
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
+ *)
+
+theory Kernel_Config_Lemmas
+imports "$L4V_ARCH/Kernel_Config"
+begin
+
+text \<open>
+  seL4's build system allows configuration of some architecture-independent constants, such as the
+  number of domains.
+
+  The long-term goal is to make the proofs resilient in the face of changes of these configuration
```

Comment:
well, for some of them, don't think we want to commit to many or most of them

Reply:
the really dumb thing is I originally wrote that :/ Fine to leave

Reply:
I'd say longer term we do want that for most of them -- at least to the degree that it becomes very easy to check if config changes are expected to pass the proofs or not. But that is for actual config parameters, not for selecting feature combinations or platform/architecture settings.

Reply:
I agree. All good.